descriptors/ModuleDescriptor-template.json

```diff
@@ -150,7 +150,7 @@
     },
     {
       "id": "circulation",
-      "version": "10.0",
+      "version": "11.1",
```

Comment:
Why has this jumped a major and minor version in one step?

Reply:
This PR was based on branch CIRC-1091, which has not been merged yet. In CIRC-1091 the version was increased to 11.0, and this PR increases it to 11.1. I'm going to merge them sequentially: CIRC-1091 and then CIRC-1143

Reply:
> This PR was based on branch CIRC-1091, which has not been merged yet. In CIRC-1091 the version was increased to 11.0, and this PR increases it to 11.1. I'm going to merge them sequentially: CIRC-1091 and then CIRC-1143

Ok, thank you for explaining that